src/prefect/cli/flow_run.py

```diff
@@ -139,13 +142,33 @@ async def cancel(id: UUID):
 
 
 @flow_run_app.command()
-async def logs(id: UUID):
+async def logs(
+    id: UUID,
+    head: bool = typer.Option(
+        False,
+        "--head",
+        "-h",
+        help=f"Show the first {LOGS_HEAD_DEFAULT_NUM_LINES} lines of logs instead of the last 10 lines",
```

Comment:
Shouldn't this be 
`f"Show the first {LOGS_HEAD_DEFAULT_NUM_LINES} lines of logs instead of all logs"` ?

Reply:
Indeed!